packages/assert/src/assert.js

```diff
@@ -61,7 +61,7 @@ harden(openDetail);
 // whereas the same diagnostic printed to the console reveals that the
 // sky was green.
 function details(template, ...args) {
-  const complainer = harden({
+  const complainer = /*harden(*/{
```

Comment:
this shouldn't still be commented out, right?

Reply:
Yes it should.  This call to `harden` was the single biggest performance suck in the whole shebang.  Fixing this will require some kind of magic refactoring of `details`.  MarkM seemed to think something like that was possible.  We should probably file and issue on that.

Reply:
Added as issue #679

Reply:
ok great